tests/integration/backend_test.py

```diff
@@ -1248,69 +1248,6 @@ def to_int(C: np.ndarray) -> int:
         assert a_count == b_count
 
 
-@pytest.mark.skipif(skip_remote_tests, reason=REASON)
-@pytest.mark.parametrize(
-    "authenticated_quum_backend_qa", [{"device_name": "H1-1E"}], indirect=True
-)
-@pytest.mark.parametrize(
-    "language",
-    [
-        Language.QIR,
-        pytest.param(
-            Language.QASM,
-            marks=pytest.mark.xfail(reason="https://github.com/CQCL/tket/issues/1174"),
-        ),
-    ],
-)
-@pytest.mark.timeout(120)
-def test_wasm_multivalue(
-    authenticated_quum_backend_qa: QuantinuumBackend, language: Language
-) -> None:
-    wasfile = WasmFileHandler(
-        str(Path(__file__).parent.parent / "wasm" / "multivalue.wasm")
```

Comment:
Can we delete this wasm file too (and the corresponding C file)?

Reply:
Yes, done in [f5a598e](https://github.com/CQCL/pytket-quantinuum/pull/418/commits/f5a598e3e4cee42c98e0670d4b84b3792f54dc80) and [9ed3e14](https://github.com/CQCL/pytket-quantinuum/pull/418/commits/9ed3e14f97c4faa28d8e91985876e3715766d039)